lib/client-api/src/config_api.ts

```diff
@@ -27,6 +27,7 @@ export default class ConfigApi {
       this._storyStore.finishConfiguring();
     } catch (err) {
       this._storyStore.setError(err);
+      this._storyStore.failConfiguring(err);
```

Comment:
Why can't we call `finishConfiguring()`? The store knows there is an error itself.